tests/testthat/test-standalone-forcats.R

```diff
@@ -54,3 +54,20 @@ test_that("fct_relevel() works", {
   f_new <- fct_relevel(f, "b", "a", "d")  # "d" is unobserved
   expect_equal(levels(f_new), c("b", "a", "d", "c"))  # "d" should be added as a level, even though not observed
 })
+
+test_that("fct_* function behaviour when input is not a factor ", {
```

Comment:
For these equality expectations, can we test against the `forcats::fct_*()` function instead?

Reply:
Sure thing! I've changed the tests to reflect that.

Reply:
Thank you!!